Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10.3-slim-buster as base
+RUN apt-get update
+COPY . /opt/
+WORKDIR /opt
```

Comment:
`/opt` might have other files in! It will probably work fine, but it's better to either use a subdirectory of opt, or pick a different folder that doesn't exist in Linux by default, like `/app`